partials/footer.html

```diff
@@ -142,7 +143,7 @@
                     </div>
                 </aside>
                 <div class="js-footer">
-                    <a href="#"><img src="/images/icon-logo-js.png" alt="icon JS" title="Javascript"></a>
+                    <a href="https://www.javascript.com/" target="_blan"> <img src="/images/icon-logo-js.png" alt="JavaScript" title="javascript.com"> </a>
```

Comment:
@robsongajunior @jaydson is this javascript.com link supposed to be here? Feels a bit out of place IMHO.